governance/overview.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Overview
+description: As a Polkadot parachain, Moonbeam will use an on-chain governance system, allowing for a stake-weighted vote on public referenda.
+---
+
+# Governance in Moonbeam
+
+![Governance Moonbeam Banner](/images/governance/governance-overview-banner.png)
+
+## Introduction
+
+Moonbeam is a decentralized network that will be governed by a community of stakeholders, including core developers, application developers, collators, token holders, users, partners, and other contributors. 
+
+We aim to facilitate the engagement of token holders from each of these categories as part of launching the network.
+
+## General Definitions
+
+With great power comes great responsibility. Some important parameters to understand before engaging with Moonbeam's governance include:
+
+ - **Proposals** — actions or items being proposed by token holders. This need to be seconded by other users in order to move to referendum
+ - **Referendum** — when the most seconded proposal is voted on by the community. There can only be one active referendum at a time unless there is an emergency referendum in progress
```

Comment:
There can be more than one referendum at a time. Check out https://github.com/PureStake/moonbeam/blob/v0.6.0/runtime/src/lib.rs#L287-L288

The launch period is one day, which means a new proposal will graduate to referendum status every day. The voting period is five days which is how long a referendum will remain open. So we can have up to 5 at a time even without emergency intervention.

Reply:
You are right, I just tested this. I got confused because of the fact that there can be only one Council referendum at a time.